### PR TITLE
add additional values for dependent income filing thresholds

### PR DESCRIPTION
**Pull request by Battula**
[ME-183396494](https://www.pivotaltracker.com/story/show/183396494)

### CODE REVIEW

File: system/config/templates/features/features.yml [hunk 1]
```diff
@@ -38,4 +38,31 @@ registry:
...
       - key: :invert_person_association
-        is_enabled: <%= ENV["INVERT_PERSON_ASSOCIATION_IS_ENABLED"] || false %>
\ No newline at end of file
+        is_enabled: <%= ENV["INVERT_PERSON_ASSOCIATION_IS_ENABLED"] || false %>
+  - namespace:
+    - :income
+    features:
+      - key: :earned_income_filing_threshold_2020
+        item: 12_400
+        is_enabled: true
```
Comment (markgoho) [inline system/config/templates/features/features.yml]: we should use environment variables here instead of hard-coded values

**Review by markgoho — CHANGES_REQUESTED** [review]

File: system/config/templates/features/features.yml (hunk 1 above)
Comment (Battula) [reply system/config/templates/features/features.yml]: @markgoho for enabling the feature or for the value ?

Comment (markgoho) [reply system/config/templates/features/features.yml]: enabling the feature would be one definite requirement -- as far as the values themselves, if these are subject to change between clients, they should also be environment variables

however, if we expect the single value to serve for all clients for an extended period of time, we could keep them as hard-coded values

Comment (markgoho) [reply system/config/templates/features/features.yml]: Just heard from Sarah B. -- she says these values are the same for DC and ME and I think we'd be okay keeping them as hard-coded values

Comment (markgoho) [reply system/config/templates/features/features.yml]: @Battula what do we gain from this layer of indirection? i.e. moving values from where they were initially set to the resource registry?

Comment (Battula) [reply system/config/templates/features/features.yml]: @markgoho Dan doesn't want hard-coded values in the system and hence introduced RR. And, it serves as one place that stores all the site level config, if that makes sense. Thanks!

Comment (markgoho) [reply system/config/templates/features/features.yml]: @Battula I would recommend reverting back to having the values directly in the `types.rb` file and not put anything in the resource registry

Comment (Battula) [reply system/config/templates/features/features.yml]: We want site level config in RR. Thanks for your recommendation though, @markgoho !

Comment (Battula) [reply system/config/templates/features/features.yml]: If the value changes the future, we can avoid code change.

Comment (markgoho) [reply system/config/templates/features/features.yml]: I'm not seeing the value in having this in the resource registry. Make the change please.

**Review by markgoho — APPROVED** [review]

**Pull request merged**